packages/clerk-js/src/ui/components/SignIn/SignIn.tsx

```diff
@@ -39,6 +39,16 @@ function SignInRoutes(): JSX.Element {
   const signUpContext = useSignUpContext();
   const options = useOptions();
 
+  if (
+    process.env.NODE_ENV === 'development' &&
+    signInContext.__experimental?.signUpProps &&
+    options.signInUrl !== signInContext.signInUrl
```

Comment:
I'm suspicious if this works given that `options.signInUrl` is the user-provided value whereas `signInContext.signInUrl` is the result of running a value through `buildUrl`, which would attach some extra stuff. Also shouldn't we be comparing a `signUpUrl` to a `signInUrl`?

Reply:
whoops https://github.com/clerk/javascript/pull/4659/commits/0b8b05621295f86abf816605f81c8e1db9a48921